includes/definitions/discovery/snr.yaml

```diff
@@ -0,0 +1,24 @@
+mib: NAG-MIB
+modules:
+    processors:
+        data:
+            -
+                  oid: sysCpuUsage
+                  num_oid: '.1.3.6.1.4.1.40418.7.100.1.8.1.11.{{ $index }}'
+    sensors:
+        temperature:
+           data:
+                -
+                    oid: sysTemperature
+                    num_oid: '.1.3.6.1.4.1.40418.7.100.1.8.1.10.{{ $index }}'
+                    descr: 'Temperature {{ $index }}'
+        state:
+           data:
+                -
```

Comment:
The states field is required for state sensors.

Reply:
it is with this model that the only thing that can be assigned to the state is the fan status and whether the ROM download is successful

Reply:
![Screenshot_50](https://user-images.githubusercontent.com/20502851/67272319-fdc7ae00-f4c4-11e9-9f44-faa1141c7f28.png)

Reply:
then you need 1 and 0

Reply:
@hrtrd I checked the code and if you don't define states, the sensor will ALWAYS show ok, no matter what the state is...  So you need to define states.